pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -139,6 +139,17 @@ type PGBackRestArchive struct {
 	// Defines details for performing an in-place restore using pgBackRest
 	// +optional
 	Restore *PGBackRestRestore `json:"restore,omitempty"`
+
+	// Defines pgBackRest sidecars
```

Comment:
:pick: Can we change the wording here to "Configuration for pgBackRest sidecars" (or similart), vs "Defines pgBackRest sidecars"?

The latter sounds like we're allowing users to define what sidecars are actually deployed (and we do actually have a separate request to be able to add custom sidecars), rather than just allowing for the configuration of certain details within the sidecars we deploy.

(similar thought would apply to PgBouncer and instances)

Reply:
:+1: That works for me, updated in all three places.